pages/common/llvm-dis.md

```diff
@@ -0,0 +1,16 @@
+# llvm-dis
+
+> Converts LLVM bitcode files into human-readable LLVM Intermediate Representation (IR).
+> More information: <https://www.llvm.org/docs/CommandGuide/llvm-dis.html>.
```

Comment:
```suggestion
> More information: <https://manned.org/llvm-dis.1>.
```

Reply:
Ok, wait for others opinions as well though

Reply:
I agree. manned.org is a _fallback_ in case an official page is not provided. In this case, we really want to keep with the llvm.org website instead, since it's an official source.